core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/IoUtil.java

```diff
@@ -136,6 +143,36 @@ public static InputStream getResource(String resource) throws IOException {
         return is;
     }
 
+    /**
+     * Lists all children resources.
+     *
+     * @param parent
+     *      the root path represented in {@link URI} format
+     * @throws java.io.IOException
+     *      if any i/o error occur
+     * @throws ProviderNotFoundException
+     *      if a provider supporting the URI scheme is not installed
+     */
+    public static void listResources(URI parent, Consumer<Path> consumer) throws IOException {
+        FileSystem fileSystem = null;
+        try {
+            if (!parent.getScheme().equals("file")) {
+                try {
+                    fileSystem = FileSystems.newFileSystem(parent, Collections.emptyMap());
```

Comment:
To get file system based on `file` scheme the URI has to be like this `file:/`. In other words `newFileSystem` methods accept the root of the file system for 'file' scheme.


So anyway I needed to add `if` clause, so I decided to omit to request file system since it exists anyway and I am able to `Paths.get(parent);` not to afraid arising  `FileSystemNotFoundException`